fix: explain file removal errors and give recommendations (#400)
vaadin/docs: articles/ds/customization/custom-theme.asciidoc

Changes to the theme folder’s contents are automatically picked up during development.
If the application has live reload enabled, the UI should reload automatically.
Otherwise a page reload may be needed to see the changes.

Sometimes, compilation throws an error (“no such file or directory”) when files or folders are removed from the theme folder while the application is running.
This might happen due to the following reasons:

> 1. The file being removed is a stylesheet referenced from `my-theme/styles.css`, or from other stylesheet in the theme folder.
> In that case, to avoid re-compilation errors, please remove the stylesheet import first, and then remove the file.
> 2. The file being removed is a component stylesheet in `my-theme/components` folder, or a whole components folder.
> The files inside components folder are referenced from auto-generated file `theme-my-theme.generated.js` and, thus, re-compilation fails, because Webpack cannot find them.

Reviewer: No need to explicitly use numbers in numbered lists.

```suggestion
. The file being removed is a stylesheet referenced from `my-theme/styles.css`, or from other stylesheet in the theme folder.
In that case, to avoid re-compilation errors, please remove the stylesheet import first, and then remove the file.
. The file being removed is a component stylesheet in `my-theme/components` folder, or a whole components folder.
The files inside components folder are referenced from auto-generated file `theme-my-theme.generated.js` and, thus, re-compilation fails, because Webpack cannot find them.
```

> To avoid errors, please remove your component stylesheet simultaneously (in one go) with the theme generated file.

Reviewer: ```suggestion
To avoid errors, remove your component style sheet simultaneously (in one go) with the theme generated file.
```

> [NOTE]
> When you encounter a “no such file or directory” error reported by Webpack and shown on the overlay in your app page, please click on the overlay (or refresh the browser page), and it should disappear, so you can continue your work with application and theming.
> If it doesn't help, please restart your app.

Reviewer: ```suggestion
[NOTE]
When you encounter a “no such file or directory” error reported by webpack and shown in the overlay in your application, click on the overlay (or refresh the browser page), and it should disappear.
You can then continue working on your application and theming.
If it doesn't help, restart need to restart your application.
```
